home_work/hw_7/bulls_and_cows_game/bulls_and_cows.py

```diff
@@ -0,0 +1,36 @@
+"""
+Module for Bulls and Cows game implementation.
+"""
+
+import random
+import string
+
+
+class BullsAndCows:
+    """Bulls and Cows game implementation."""
+
+    def __init__(self):
+        """Initializes a new instance of the BullsAndCows class."""
+        self.secret_number = self.gen_four_digit_num()
```

Comment:
If are not planning to generate random numbers more than ones, so let's move it here:
`self.secret_number = "".join(random.sample(string.digits, 4))`

Reply:
moved
![image](https://github.com/Eystei/pymar2024/assets/95315504/9ee1ec88-b89b-454a-a3f0-da7c181fa45d)